corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -49,25 +58,29 @@ def _should_sync(restore_state):
     return (
         not last_sync_log or
         not sync_interval or
-        (datetime.utcnow() - last_sync_log.date).total_seconds() > sync_interval
+        (_utcnow() - last_sync_log.date).total_seconds() > sync_interval
     )
 
 
-class ReportFixturesProvider(FixtureProvider):
-    id = 'commcare:reports'
-
-    def __call__(self, restore_state):
-        """
-        Generates a report fixture for mobile that can be used by a report module
-        """
+class BaseReportFixturesProvider(FixtureProvider):
+    def uses_reports(self, restore_state):
         restore_user = restore_state.restore_user
         if not toggles.MOBILE_UCR.enabled(restore_user.domain) or not _should_sync(restore_state):
-            return []
+            return False
 
         if toggles.PREVENT_MOBILE_UCR_SYNC.enabled(restore_user.domain):
-            return []
+            return False
+
+        apps = self._get_apps(restore_state, restore_user)
```

Comment:
Any concern that this will get called twice for each restore?

Reply:
With app-aware sync, this will get it from the restore_state https://github.com/dimagi/commcare-hq/blob/b1b4a5636cacba02a7925799deb22307a72afe74/corehq/apps/app_manager/fixtures/mobile_ucr.py#L78 

Clayton said that web apps now supports app aware sync (at least with enikshay), so I think this should be fine